text/0000-trait-alias.md

```diff
@@ -0,0 +1,266 @@
+- Feature Name: Trait alias
+- Start Date: 2016-08-31
+- RFC PR:
+- Rust Issue:
+
+# Summary
+[summary]: #summary
+
+Traits can be aliased the same way types can be aliased with the `type` keyword.
```

Comment:
The initial intention was to state that we can have *alias* the same way we already have with the `type` keyword and types. But you’re right, it’s confusing. I’m updating that.